docs/topics/testing/tools.txt

```diff
@@ -2136,6 +2136,40 @@ and contents::
             # Verify that the subject of the first message is correct.
             self.assertEqual(mail.outbox[0].subject, "Subject here")
 
+Here is another example testing an email with
+:ref:`multiple content types <topic-alternative-content-types>`::
```

Comment:
@sarahboyce There seems to be an issue with that link... Not 100% sure how this works, though 😅